### PR TITLE
12 delete comments

### PR DESCRIPTION
**Pull request by kalumwildig**

### CODE REVIEW

File: controllers/controller.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ const {
   getUserModel,
   getArticlesModel,
   getArticleCommentsModel,
+
+  deleteCommentModel,
+  getCommentModel
```
Comment (douglashellowell) [inline controllers/controller.js:10]: there's a syntax error here - you're missing a `,`!

i'm seeing a syntax error in your `app.test.js` too
Not sure if this is github being weird or if you missed it before commiting...?

File: controllers/controller.js [hunk 2]
```diff
@@ -56,6 +60,16 @@ exports.getArticleComments = async (req, res, next) => {
   }
 };
 
+exports.deleteComment = async (req, res, next) => {
+   try { const id = req.params.comment_id
+    await Promise.all([getCommentModel(id),deleteCommentModel(id)])
```
Comment (rogersop) [inline controllers/controller.js:65]: Slight overkill using your getCommentModel here, as the result object from the deleteCommentModel will let you know how many rows you have deleted with your query, which can be used to let the client know that either:
- the comment was successfully deleted, or
- the comment was not found

**Review by rogersop — CHANGES_REQUESTED** [review]
See both mine and @douglashellowell's comments - we both started looking at the same time! 😅

**Review by douglashellowell — CHANGES_REQUESTED** [review]
see both Paul and my own comments

have a check you haven't got some syntax errors in your code

Also, if you haven't got prettier installed please do!

File: controllers/controller.js [hunk 3]
```diff
@@ -64,6 +66,16 @@ exports.getArticleComments = async (req, res, next) => {
   }
 };
 
+exports.deleteComment = async (req, res, next) => {
+   try { const id = req.params.comment_id
```
Comment (Augs0) [inline controllers/controller.js:70]: Just an organisational note, try to make sure your code is on new lines for readability :)

**Review by Augs0 — APPROVED** [review]

**Pull request merged**